### PR TITLE
update mobile release schedule

### PR DESCRIPTION
**Pull request by mandrigin**
more releases, mark released versions

### CODE REVIEW

File: source/mobile-release-schedule/index.md [hunk 1]
```diff
@@ -37,6 +38,26 @@ var releases = [
...
+        dates: [new Date(2019, 6, 2)],
+        title: "Release 0.17.0",
+    },
+    {
+        dates: [new Date(2019, 6, 16)],
+        title: "Release 0.18.0",
+    },
+    {
+        dates: [new Date(2019, 6, 30)],
+        title: "Release 1.0.0",
```
Comment (jakubgs) [inline source/mobile-release-schedule/index.md]: Oh snap.

**Comment by jakubgs** [comment]
Where is this page actually visible? I can't find it on https://dev.status.im/

**Comment by mandrigin** [comment]
@jakubgs https://dev.status.im/mobile-release-schedule/ but I don't see it updated

**Comment by jakubgs** [comment]
It is now. There's something wrong with https://ci.status.im/job/misc/job/dev.status.im/ because it doesn't run on new commits in `develop` branch. Gonna check that out.

**Pull request merged**